eng/Version.Details.xml

```diff
@@ -1,9 +1,5 @@
 <Dependencies>
   <ProductDependencies>
-    <Dependency Name="NETStandard.Library" Version="2.2.0-prerelease.19564.1">
-      <Uri>https://github.com/dotnet/standard</Uri>
-      <Sha>cfe95a23647c7de1fe1a349343115bd7720d6949</Sha>
-    </Dependency>
```

Comment:
@dseefeld do we need to keep the dependency entry here for sourcebuild?

Reply:
> @dseefeld Dan Seefeldt FTE do we need to keep the dependency entry here for sourcebuild?

No.  That shouldn't be needed for source-build.

Reply:
Great, just wanted to make sure :)